app/src/main/java/de/tum/in/tumcampusapp/adapters/FeedbackThumbAdapter.java

```diff
@@ -0,0 +1,122 @@
+package de.tum.in.tumcampusapp.adapters;
+
+import android.app.AlertDialog;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.net.Uri;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+
+import java.io.File;
+import java.util.ArrayList;
+
+import de.tum.in.tumcampusapp.R;
+
+public class FeedbackThumbAdapter extends RecyclerView.Adapter<FeedbackThumbAdapter.ViewHolder> {
+    private ArrayList<String> paths;
+    private ArrayList<Bitmap> thumbs;
+
+    private int viewing;
+
+    public static class ViewHolder extends RecyclerView.ViewHolder {
+        private ImageView imageView;
+        public ViewHolder(ImageView img){
+            super(img);
+            imageView = img;
+        }
+    }
+
+    public FeedbackThumbAdapter(ArrayList<String> paths){
+        this.paths = paths;
+        thumbs = new ArrayList<>(paths.size());
+    }
+
+    @Override
+    public ViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        ImageView imageView = (ImageView) LayoutInflater.from(parent.getContext())
+                                                        .inflate(R.layout.feedback_thumb, parent, false);
+        return new ViewHolder(imageView);
+    }
+
+    @Override
+    public void onBindViewHolder(ViewHolder holder, int position) {
+        if(thumbs.size() < position+1 || thumbs.get(position) == null){
+            Bitmap bitmap = createThumb(holder.imageView, position);
```

Comment:
`createThumb()` can return `null`. Shouldn't this be handled?

Reply:
I'm now deleting images for which I can't create a thumbnail